pkg/platform/model/vcs.go

```diff
@@ -643,6 +600,38 @@ func CommitInitial(hostPlatform string, langName, langVersion string) (strfmt.UU
 	return res.Payload.CommitID, nil
 }
 
+func isWildcardVersion(version string) bool {
+	return strings.Index(version, "x") >= 0 || strings.Index(version, "X") >= 0
```

Comment:
For `init` you require `x` to be after a dot. Here you don't. I think requiring the dot probably makes more sense, suggests we keep it consistent.